guillotina/tests/fixtures.py

```diff
@@ -499,10 +499,9 @@ def mock_txn():
 
 
 @pytest.fixture(scope="function")
-def guillotina(app_client):
+async def guillotina(app_client):
     _, client = app_client
-    requester = GuillotinaDBAsgiRequester(client)
-    yield requester
+    return GuillotinaDBAsgiRequester(client)
```

Comment:
This change + https://github.com/plone/guillotina_elasticsearch/compare/3.3.x...masipcat:3.3.x-g6 makes guillotina_elasticsearch compatible with G6!